api/storage/elasticsearch/elasticsearch.go

```diff
@@ -331,6 +325,45 @@ func (sm *StorageManager) GetResources(resourceType string, executionID string)
 	return resources, nil
 }
 
+// GetExecutionTags will return the tags according to a given executionID
+func (sm *StorageManager) GetExecutionTags(executionID string) (map[string][]string, error) {
+
+	tags := map[string][]string{}
+	eventTypeMatchQuery := elastic.NewMatchQuery("EventType", "resource_detected")
+	executionIDMatchQuery := elastic.NewMatchQuery("ExecutionID", executionID)
+	elasticQuery := elastic.Query(elastic.NewBoolQuery().Must(eventTypeMatchQuery, executionIDMatchQuery))
+
+	searchResult, err := sm.client.Search().
+		Query(elasticQuery).
+		Pretty(true).
+		Size(100).
+		Do(context.Background())
+
+	if err != nil {
+		log.WithError(err).Error("got an elasticsearch error while running the query")
+		return tags, err
+	}
+	var availableTags TagsData
+	for _, hit := range searchResult.Hits.Hits {
+
+		err := json.Unmarshal([]byte(string(hit.Source)), &availableTags)
+		if err != nil {
+			log.WithError(err).Error("error when trying to parse tags map")
```

Comment:
Are you not handle this error properly.
do you want to stop the loop? continue?

Reply:
Added Continue.